src/js/controller/instream-html5.js

```diff
@@ -92,6 +92,7 @@ define([
             if (_currentProvider) {
                 _currentProvider.detachMedia();
                 _currentProvider.off();
+                _currentProvider.off(null, null, _this);
```

Comment:
`_currentProvider.off()` should have removed all event listeners. Why do that and `_currentProvider.off(null, null, _this)`?